ngraph/python/src/ngraph/utils/node_factory.py

```diff
@@ -40,6 +40,13 @@ def create(
         if attributes is None:
             attributes = {}
 
+        if attributes is None and arguments is None:
```

Comment:
What happens if attributes are present but arguments are `None`? That is one case that should be somehow supported.

Reply:
Moreover if attributes are `None` then lines 40-41 are setting them to the empty dictionary. This line will never be executed since attributes are going not equal to `None`. This check should be first one and after that there should be setting of the empty dictionary.

Reply:
Fixed.